apps/yapms/src/lib/components/sidebar/SideBar.svelte

```diff
@@ -25,5 +25,8 @@
 		</div>
 		<h1 class="text-xl text-center font-bold">{title}</h1>
 		<div class="divider">Shortcuts</div>
+		{#if $PocketBaseStore.authStore.isValid}
```

Comment:
Instead of completely hiding this section if you are not logged in, maybe we could have it display a message like "Login to save maps to your account" w/ a login button below.

Reply:
I'd rather not have two login buttons on the page at the same time. Maybe there could be another issue opened to have the login process moved to the sidebar on desktop.

Reply:
I would still think that a notice somewhere that this feature exists to non-logged in people would be good at some point, but it doesn't have to be in the sidebar.